app/client/src/routes/home.tsx

```diff
@@ -100,61 +209,76 @@ function getInputValue(input: Exclude<InputState[keyof InputState], null>) {
   return input;
 }
 
+function getLocalStorageItem(item: string) {
+  if (storageAvailable()) {
+    return localStorage.getItem(item) ?? null;
+  } else return null;
+}
```

Comment:
```suggestion
function getLocalStorageItem(item: string) {
  return localStorage.getItem(item) ?? null;
}
```

The call to `storageAvailable` is causing the `__storage_test__` item to be written and removed from local storage on every render. I think we should just remove the call to `storageAvailable` as we shouldn't need to check wether storage is available to get an item out of local storage.

Reply:
Good point, I'll change that.